include/quda.h

```diff
@@ -1658,6 +1708,19 @@ extern "C" {
    */
   void performWuppertalnStep(void *h_out, void *h_in, QudaInvertParam *param, unsigned int n_steps, double alpha);
 
+  /**
```

Comment:
We should make these "legacy" functions thin wrappers around `performFermionSmearQuda`. Nothing that needs to be done for this PR, but we should at least file an issue about it.

Reply:
https://github.com/lattice/quda/issues/1474